test/integration/rest-basic/index.spec.ts

```diff
@@ -41,14 +44,6 @@ test('rest basics', async ({ page, context, localApp }) => {
 
   await expect(page.getByText('query4 authorization: foo')).toBeVisible();
 
-  const envFilePath = path.resolve(localApp.dir, './.env');
```

Comment:
Why was this removed? This is testing whether the editor detects changes to the env file correctly

Reply:
Dotenv wasn't working with tests locally, I'll try to make it work though so we can test like this.

Reply:
Both methods need to work. The key is to add a second env var so that the old test remains as it is. Like it's done for https://github.com/mui/mui-toolpad/pull/2103/files#diff-f16a8b4fc36e8dc38f1edc79e02791f23fed2ae1be7d7f7b1b015c1a05aec6ebR23-R25